src/include/local/stir/motion/NonRigidObjectTransformationUsingBSplines.h

```diff
@@ -53,15 +53,25 @@ class NonRigidObjectTransformationUsingBSplines
 public:
   static const char * const registered_name;
 
+  // Default constructor
   NonRigidObjectTransformationUsingBSplines();
 
+  // Give x, y and z components of the deformation field images separately
+  NonRigidObjectTransformationUsingBSplines(const std::string, const std::string, const std::string, int &bspline_order);
+
+  // Give x, y and z components of the deformation field images together (e.g., multicomponent nifti)
+  NonRigidObjectTransformationUsingBSplines(const std::string, int &bspline_order);
+
   //! Transform point 
   virtual
     BasicCoordinate<num_dimensions,elemT>
     transform_point(const BasicCoordinate<num_dimensions,elemT>& point) const;
 
   float jacobian(const BasicCoordinate<num_dimensions,elemT>& point) const;
 
+  /// Set deformation field (dimension=1,2,3 for x,y,z)
+  Succeeded set_deformation_field(const std::string &filename, const int &dimension);
```

Comment:
no reason to use `const int&`. it'll actually be a tiny bit slower. just use `const int`